vagrant/provisioning/roles/foia/templates/application.properties

```diff
@@ -41,15 +41,15 @@ portal.truststoreFilename=arkcase.ts
 portal.truststorePassword={{ java_trust_store_pass }}
 
 # authentication type, possible values (basic, external)
-portal.authenticationType=basic
+portal.authenticationType={{ foia_portal_authentication_type | default('basic') }}
 # basic
 portal.authenticationBasicUser={{ arkcase_admin_user }}@{{ ldap_user_domain }}
 portal.authenticationBasicPassword={{ arkcase_admin_password }}
 # external
 portal.authenticationExternalUser={{ arkcase_admin_user }}@{{ ldap_user_domain }}
 
 # ArkCase url
-portal.arkcaseUrl=https://{{ internal_host }}/arkcase
+portal.arkcaseUrl=https://{{ internal_host }}:8843/arkcase
```

Comment:
This one should work through the proxy without a port.

Reply:
In multi-node environments the proxy isn't local.  I could have tried using external_host versus internal_host, and maybe I'll see if that will work next.  This way seemed easier, and avoids a round trip to the proxy.  Of course if the portal ever ends up on a different host than ArkCase we'll need to use external_host anyway.